### ISSUE TITLE
[mpfr] Build error

### ISSUE THREAD

**Issue by archibate** [microsoft/vcpkg#24286]
**Host Environment**
- OS: Linux
- Compiler: GCC 9

**To Reproduce**[
`./vcpkg install mpfr`
See this workflow log: https://github.com/zenustech/zeno/runs/6105211406?check_suite_focus=true

**Failure logs**
<details>

```
Install/Update ports using vcpkg.json
  Running 'vcpkg install,--recurse,--clean-after-build,--x-install-root,/home/runner/work/zeno/b/vcpkg_installed,--triplet,x64-linux' in directory '/home/runner/work/zeno/zeno' ...
  Running command '/home/runner/work/zeno/zeno/vcpkg/vcpkg' with args 'install,--recurse,--clean-after-build,--x-install-root,/home/runner/work/zeno/b/vcpkg_installed,--triplet,x64-linux' in current directory '/home/runner/work/zeno/zeno'.
  Detecting compiler hash for triplet x64-linux...
  The following packages will be built and installed:
      alembic[core,hdf5]:x64-linux -> 1.8.3#1
    * blas[core]:x64-linux -> 1#1
    * blosc[core]:x64-linux -> 1.18.1#4
    * boost-accumulators[core]:x64-linux -> 1.78.0
    * boost-algorithm[core]:x64-linux -> 1.78.0
    * boost-align[core]:x64-linux -> 1.78.0
    * boost-any[core]:x64-linux -> 1.78.0
    * boost-array[core]:x64-linux -> 1.78.0
    * boost-assert[core]:x64-linux -> 1.78.0
    * boost-atomic[core]:x64-linux -> 1.78.0
    * boost-bimap[core]:x64-linux -> 1.78.0
    * boost-bind[core]:x64-linux -> 1.78.0
    * boost-build[core]:x64-linux -> 1.78.0#1
    * boost-callable-traits[core]:x64-linux -> 1.78.0
    * boost-chrono[core]:x64-linux -> 1.78.0
    * boost-circular-buffer[core]:x64-linux -> 1.78.0
    * boost-compute[core]:x64-linux -> 1.78.0
    * boost-concept-check[core]:x64-linux -> 1.78.0
    * boost-config[core]:x64-linux -> 1.78.0
    * boost-container[core]:x64-linux -> 1.78.0
    * boost-container-hash[core]:x64-linux -> 1.78.0
    * boost-conversion[core]:x64-linux -> 1.78.0
    * boost-core[core]:x64-linux -> 1.78.0
    * boost-date-time[core]:x64-linux -> 1.78.0
    * boost-detail[core]:x64-linux -> 1.78.0
    * boost-dynamic-bitset[core]:x64-linux -> 1.78.0
    * boost-endian[core]:x64-linux -> 1.78.0
    * boost-exception[core]:x64-linux -> 1.78.0
    * boost-filesystem[core]:x64-linux -> 1.78.0
    * boost-foreach[core]:x64-linux -> 1.78.0
    * boost-format[core]:x64-linux -> 1.78.0
    * boost-function[core]:x64-linux -> 1.78.0
    * boost-function-types[core]:x64-linux -> 1.78.0
    * boost-functional[core]:x64-linux -> 1.78.0
    * boost-fusion[core]:x64-linux -> 1.78.0
    * boost-geometry[core]:x64-linux -> 1.78.0
    * boost-graph[core]:x64-linux -> 1.78.0
    * boost-heap[core]:x64-linux -> 1.78.0
    * boost-integer[core]:x64-linux -> 1.78.0
    * boost-interprocess[core]:x64-linux -> 1.78.0
    * boost-interval[core]:x64-linux -> 1.78.0
    * boost-intrusive[core]:x64-linux -> 1.78.0
    * boost-io[core]:x64-linux -> 1.78.0
    * boost-iostreams[bzip2,core,lzma,zlib,zstd]:x64-linux -> 1.78.0#2
    * boost-iterator[core]:x64-linux -> 1.78.0
    * boost-lambda[core]:x64-linux -> 1.78.0
    * boost-lexical-cast[core]:x64-linux -> 1.78.0
    * boost-logic[core]:x64-linux -> 1.78.0
    * boost-math[core]:x64-linux -> 1.78.0
    * boost-modular-build-helper[core]:x64-linux -> 1.78.0#3
    * boost-move[core]:x64-linux -> 1.78.0
    * boost-mp11[core]:x64-linux -> 1.78.0
    * boost-mpl[core]:x64-linux -> 1.78.0
    * boost-multi-index[core]:x64-linux -> 1.78.0
    * boost-multiprecision[core]:x64-linux -> 1.78.0
    * boost-numeric-conversion[core]:x64-linux -> 1.78.0
    * boost-optional[core]:x64-linux -> 1.78.0
    * boost-parameter[core]:x64-linux -> 1.78.0
    * boost-phoenix[core]:x64-linux -> 1.78.0
    * boost-polygon[core]:x64-linux -> 1.78.0
    * boost-pool[core]:x64-linux -> 1.78.0
    * boost-predef[core]:x64-linux -> 1.78.0
    * boost-preprocessor[core]:x64-linux -> 1.78.0
    * boost-property-map[core]:x64-linux -> 1.78.0
    * boost-property-tree[core]:x64-linux -> 1.78.0
    * boost-proto[core]:x64-linux -> 1.78.0
    * boost-ptr-container[core]:x64-linux -> 1.78.0
    * boost-qvm[core]:x64-linux -> 1.78.0
    * boost-random[core]:x64-linux -> 1.78.0
    * boost-range[core]:x64-linux -> 1.78.0
    * boost-ratio[core]:x64-linux -> 1.78.0
    * boost-rational[core]:x64-linux -> 1.78.0
    * boost-regex[core]:x64-linux -> 1.78.0
    * boost-serialization[core]:x64-linux -> 1.78.0
    * boost-smart-ptr[core]:x64-linux -> 1.78.0
    * boost-spirit[core]:x64-linux -> 1.78.0
    * boost-static-assert[core]:x64-linux -> 1.78.0
    * boost-system[core]:x64-linux -> 1.78.0
    * boost-thread[core]:x64-linux -> 1.78.0
    * boost-throw-exception[core]:x64-linux -> 1.78.0
    * boost-tokenizer[core]:x64-linux -> 1.78.0
    * boost-tti[core]:x64-linux -> 1.78.0
    * boost-tuple[core]:x64-linux -> 1.78.0
    * boost-type-index[core]:x64-linux -> 1.78.0
    * boost-type-traits[core]:x64-linux -> 1.78.0
    * boost-typeof[core]:x64-linux -> 1.78.0
    * boost-ublas[core]:x64-linux -> 1.78.0
    * boost-uninstall[core]:x64-linux -> 1.78.0
    * boost-units[core]:x64-linux -> 1.78.0
    * boost-unordered[core]:x64-linux -> 1.78.0
    * boost-utility[core]:x64-linux -> 1.78.0
    * boost-uuid[core]:x64-linux -> 1.78.0
    * boost-variant[core]:x64-linux -> 1.78.0
    * boost-variant2[core]:x64-linux -> 1.78.0
    * boost-vcpkg-helpers[core]:x64-linux -> 1.78.0#1
    * boost-winapi[core]:x64-linux -> 1.78.0
    * boost-xpressive[core]:x64-linux -> 1.78.0
    * bzip2[core]:x64-linux -> 1.0.8#2
      cgal[core]:x64-linux -> 5.4
      eigen3[core]:x64-linux -> 3.4.0#2
    * gmp[core]:x64-linux -> 6.2.1#9
    * hdf5[core,szip,zlib]:x64-linux -> 1.12.1#4
    * ilmbase[core]:x64-linux -> 2.3.0#1
    * imath[core]:x64-linux -> 3.1.5
      lapack[core]:x64-linux -> 2022-02-22
    * lapack-reference[blas-select,core,noblas]:x64-linux -> 3.10.0
    * liblzma[core]:x64-linux -> 5.2.5#5
    * lz4[core]:x64-linux -> 1.9.3#3
    * mpfr[core]:x64-linux -> 4.1.0#4
      openblas[core]:x64-linux -> 0.3.19
    * openexr[core]:x64-linux -> 2.5.0#3
      openvdb[core]:x64-linux -> 9.0.0
    * pthread[core]:x64-linux -> 3.0.0#1
    * pthreads[core]:x64-linux -> 3.0.0#10
    * snappy[core]:x64-linux -> 1.1.9#1
    * szip[core]:x64-linux -> 2.1.1#7
      tbb[core]:x64-linux -> 2020_U3#8
    * vcpkg-cmake[core]:x64-linux -> 2022-04-12
    * vcpkg-cmake-config[core]:x64-linux -> 2022-02-06
    * zlib[core]:x64-linux -> 1.2.12
    * zstd[core]:x64-linux -> 1.5.2#1
  Additional packages (*) will be modified to complete this operation.
  Restored 0 packages from /home/runner/work/zeno/b/vcpkg_cache in 268.5 us. Use --debug to see more details.
  Starting package 1/121: zlib:x64-linux
  Building package zlib[core]:x64-linux...
  -- Downloading https://www.zlib.net/zlib-1.2.12.tar.gz -> zlib1212.tar.gz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/zlib1212.tar.gz
  -- Applying patch cmake_dont_build_more_than_needed.patch
  -- Applying patch 0001-Prevent-invalid-inclusions-when-HAVE_-is-set-to-0.patch
  -- Applying patch add_debug_postfix_on_mingw.patch
  -- Applying patch 0002-android-build-mingw.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/zlib/src/1.2.12-220a70ae9f.clean
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Building x64-linux-rel
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/zlib_x64-linux/share/zlib/vcpkg-cmake-wrapper.cmake
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/zlib_x64-linux/lib/pkgconfig/zlib.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/zlib_x64-linux/debug/lib/pkgconfig/zlib.pc
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/zlib_x64-linux/share/zlib/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/0a/0acded0b6241f8957380fe13aa36e26a0e612af9330a12883775b2cf6d5313a3.zip
  Installing package zlib[core]:x64-linux...
  Elapsed time for package zlib:x64-linux: 4.223 s
  Starting package 2/121: vcpkg-cmake-config:x64-linux
  Building package vcpkg-cmake-config[core]:x64-linux...
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake-config_x64-linux/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake-config_x64-linux/share/vcpkg-cmake-config/vcpkg-port-config.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake-config_x64-linux/share/vcpkg-cmake-config/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/cb/cb2384448d03516802117be8281550aeab880731f77cb5f07bc126bcd48a2d7b.zip
  Installing package vcpkg-cmake-config[core]:x64-linux...
  Elapsed time for package vcpkg-cmake-config:x64-linux: 25 ms
  Starting package 3/121: vcpkg-cmake:x64-linux
  Building package vcpkg-cmake[core]:x64-linux...
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/vcpkg_cmake_configure.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/vcpkg_cmake_get_vars.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/cmake_get_vars
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/cmake_get_vars/CMakeLists.txt
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/vcpkg-port-config.cmake
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/vcpkg-cmake_x64-linux/share/vcpkg-cmake/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/6f/6f2c83d030a59ef9bb96[406](https://github.com/zenustech/zeno/runs/6105211406?check_suite_focus=true#step:5:406)63f48a5fce2567b5bbecf2838fbcf1f655b6c9252.zip
  Installing package vcpkg-cmake[core]:x64-linux...
  Elapsed time for package vcpkg-cmake:x64-linux: 24.13 ms
  Starting package 4/121: szip:x64-linux
  Building package szip[core]:x64-linux...
  -- Downloading https://support.hdfgroup.org/ftp/lib-external/szip/2.1.1/src/szip-2.1.1.tar.gz -> szip-2.1.1.tar.gz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/szip-2.1.1.tar.gz
  -- Applying patch fix-linkage-config.patch
  -- Applying patch mingw-lib-names.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/szip/src/szip-2.1.1-15a12784d5.clean
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Building x64-linux-rel
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/szip_x64-linux/lib/pkgconfig/szip.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/szip_x64-linux/debug/lib/pkgconfig/szip.pc
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/3f/3f70f54f669799caa1e07bc750e12dcf0f3ec57529cdf79fb4fb5f9c6558c4ac.zip
  Installing package szip[core]:x64-linux...
  Elapsed time for package szip:x64-linux: 12.43 s
  Starting package 5/121: hdf5:x64-linux
  Building package hdf5[core,szip,zlib]:x64-linux...
  -- Downloading https://github.com/HDFGroup/hdf5/archive/hdf5-1_12_1.tar.gz -> HDFGroup-hdf5-hdf5-1_12_1.tar.gz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/HDFGroup-hdf5-hdf5-1_12_1.tar.gz
  -- Applying patch hdf5_config.patch
  -- Applying patch szip.patch
  -- Applying patch pkgconfig-requires.patch
  -- Applying patch pkgconfig-link-order.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/hdf5/src/df5-1_12_1-859faf3d27.clean
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Building x64-linux-rel
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/hdf5_x64-linux/lib/pkgconfig/hdf5.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/hdf5_x64-linux/lib/pkgconfig/hdf5_hl.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/hdf5_x64-linux/debug/lib/pkgconfig/hdf5.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/hdf5_x64-linux/debug/lib/pkgconfig/hdf5_hl.pc
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/29/29162842b47e15b13b94895b5a6e4974df2e861870dd649fd35a55a8c6ce04eb.zip
  Installing package hdf5[core,szip,zlib]:x64-linux...
  Elapsed time for package hdf5:x64-linux: 2.348 min
  Starting package 6/121: imath:x64-linux
  Building package imath[core]:x64-linux...
  -- Downloading https://github.com/AcademySoftwareFoundation/Imath/archive/v3.1.5.tar.gz -> AcademySoftwareFoundation-Imath-v3.1.5.tar.gz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/AcademySoftwareFoundation-Imath-v3.1.5.tar.gz
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/imath/src/v3.1.5-4b13c688b3.clean
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Building x64-linux-rel
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/imath_x64-linux/lib/pkgconfig/Imath.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/imath_x64-linux/debug/lib/pkgconfig/Imath.pc
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/imath_x64-linux/share/imath/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/93/93ee733c62b540ea535b8fdee877f8208fafdab05726f13bad377c26d0d6dc4b.zip
  Installing package imath[core]:x64-linux...
  Elapsed time for package imath:x64-linux: 6.633 s
  Starting package 7/121: alembic:x64-linux
  Building package alembic[core,hdf5]:x64-linux...
  -- Downloading https://github.com/alembic/alembic/archive/1.8.3.tar.gz -> alembic-alembic-1.8.3.tar.gz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/alembic-alembic-1.8.3.tar.gz
  -- Applying patch fix-runtime-destination.patch
  -- Applying patch disable-warnings-as-error.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/alembic/src/1.8.3-d40d01fd6a.clean
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Building x64-linux-rel
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/alembic_x64-linux/share/alembic/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/65/652e1f673aa9f0e6de048d236f7b139c28b5868db0ce398eb1a08b1b839f10fd.zip
  Installing package alembic[core,hdf5]:x64-linux...
  Elapsed time for package alembic:x64-linux: 5.344 min
  Starting package 8/121: gmp:x64-linux
  Building package gmp[core]:x64-linux...
  -- Downloading https://gmplib.org/download/gmp/gmp-6.2.1.tar.xz -> gmp-6.2.1.tar.xz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/gmp-6.2.1.tar.xz
  -- Applying patch tools.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/gmp/src/gmp-6.2.1-f412680bba.clean
  -- Getting CMake variables for x64-linux-dbg
  -- Getting CMake variables for x64-linux-rel
  -- Generating configure for x64-linux
  -- Finished generating configure for x64-linux
  -- Configuring x64-linux-dbg
  -- Configuring x64-linux-rel
  -- Building x64-linux-dbg
  -- Installing x64-linux-dbg
  -- Building x64-linux-rel
  -- Installing x64-linux-rel
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/gmp_x64-linux/lib/pkgconfig/gmp.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/gmp_x64-linux/lib/pkgconfig/gmpxx.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/gmp_x64-linux/debug/lib/pkgconfig/gmp.pc
  -- Fixing pkgconfig file: /home/runner/work/zeno/zeno/vcpkg/packages/gmp_x64-linux/debug/lib/pkgconfig/gmpxx.pc
  -- Installing: /home/runner/work/zeno/zeno/vcpkg/packages/gmp_x64-linux/share/gmp/copyright
  -- Performing post-build validation
  -- Performing post-build validation done
  Stored binary cache: /home/runner/work/zeno/b/vcpkg_cache/8a/8a8c5c004bccc12be1f999e67560f8ebda54f0ea2bb70f139454832529641bb9.zip
  Installing package gmp[core]:x64-linux...
  Elapsed time for package gmp:x64-linux: 1.951 min
  Starting package 9/121: mpfr:x64-linux
  Building package mpfr[core]:x64-linux...
  CMake Warning at ports/mpfr/portfile.cmake:2 (message):
  Warning:   mpfr currently requires the following packages:
  
        autoconf-archive
  
    These can be installed on Ubuntu systems via
  
        sudo apt-get update -y
        sudo apt-get install -y autoconf-archive
  
  Call Stack (most recent call first):
    scripts/ports.cmake:145 (include)
  
  
  -- Downloading http://www.mpfr.org/mpfr-4.1.0/mpfr-4.1.0.tar.xz;https://ftp.gnu.org/gnu/mpfr/mpfr-4.1.0.tar.xz -> mpfr-4.1.0.tar.xz...
  -- Extracting source /home/runner/work/zeno/zeno/vcpkg/downloads/mpfr-4.1.0.tar.xz
  -- Applying patch dll.patch
  -- Applying patch src-only.patch
  -- Using source at /home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/src/mpfr-4-2533b4c96d.clean
  -- Getting CMake variables for x64-linux-dbg
  -- Getting CMake variables for x64-linux-rel
  -- Generating configure for x64-linux
  [LogCollection][Start]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-out.log':
  libtoolize: putting auxiliary files in '.'.
  libtoolize: copying file './ltmain.sh'
  libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
  libtoolize: copying file 'm4/libtool.m4'
  libtoolize: copying file 'm4/ltoptions.m4'
  libtoolize: copying file 'm4/ltsugar.m4'
  libtoolize: copying file 'm4/ltversion.m4'
  libtoolize: copying file 'm4/lt~obsolete.m4'
  
  [LogCollection][End]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-out.log'.
  [LogCollection][Start]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-err.log':
  autoreconf: Entering directory `.'
  autoreconf: configure.ac: not using Gettext
  autoreconf: running: aclocal --force -I m4
  aclocal: warning: couldn't open directory 'm4': No such file or directory
  autoreconf: configure.ac: tracing
  autoreconf: running: libtoolize --copy --force
  autoreconf: running: /usr/bin/autoconf --force
  configure:14306: error: possibly undefined macro: AX_PTHREAD
        If this token and others are legitimate, please use m4_pattern_allow.
        See the Autoconf documentation.
  autoreconf: /usr/bin/autoconf failed with exit status: 1
  
  [LogCollection][End]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-err.log'.
  [LogCollection][Start]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-out.log':
  libtoolize: putting auxiliary files in '.'.
  libtoolize: copying file './ltmain.sh'
  libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
  libtoolize: copying file 'm4/libtool.m4'
  libtoolize: copying file 'm4/ltoptions.m4'
  libtoolize: copying file 'm4/ltsugar.m4'
  libtoolize: copying file 'm4/ltversion.m4'
  libtoolize: copying file 'm4/lt~obsolete.m4'
  
  [LogCollection][End]File:'/home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-out.log'.
  CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:128 (message):
  Error:     Command failed: /usr/bin/autoreconf -vfi
      Working Directory: /home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/src/mpfr-4-2533b4c96d.clean/
      Error code: 1
      See logs for more information:
        /home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-out.log
        /home/runner/work/zeno/zeno/vcpkg/buildtrees/mpfr/autoconf-x64-linux-err.log
  
  Call Stack (most recent call first):
    scripts/cmake/vcpkg_configure_make.cmake:629 (vcpkg_execute_required_process)
    ports/mpfr/portfile.cmake:21 (vcpkg_configure_make)
    scripts/ports.cmake:145 (include)
  
  
  error: building mpfr:x64-linux failed with: BUILD_FAILED
  Please ensure you're using the latest portfiles with `git pull` and `./vcpkg update`.
  Then check for known issues at:
    https://github.com/microsoft/vcpkg/issues?q=is%3Aissue+is%3Aopen+in%3Atitle+mpfr
  You can submit a new issue at:
    https://github.com/microsoft/vcpkg/issues/new?template=report-package-build-failure.md&title=[mpfr]+Build+error
  including:
    package: mpfr[core]:x64-linux -> 4.1.0#4
      vcpkg-tool version: 2022-03-30-692785ac944e81417840c6de244fb3e18a4b35eb
      vcpkg-scripts version: e0c3[408](https://github.com/zenustech/zeno/runs/6105211406?check_suite_focus=true#step:5:408)c7 [202](https://github.com/zenustech/zeno/runs/6105211406?check_suite_focus=true#step:5:202)2-04-20 (4 hours ago)
```
</details>
**Additional context**
Add any other context about the problem here, such as what you have already tried to resolve the issue.

**Comment by archibate** [comment]
I see #23641 says:
```
        sudo apt-get update -y
        sudo apt-get install -y autoconf-archive
```
Why not install this `autoconf-archive` as a dep of mpfr using vcpkg?

**Comment by LilyWangLL** [comment]
Thanks for posting this issue. `autoconf-archive` is a Linux system library, vcpkg did not add this library to it。

**Comment by archibate** [comment]
But vcpkg can even download `powershell` automatically on win... Is there any difficulty on making `autoconf-archive` self-contained? Anyway, I have fixed this by special treating Ubuntu 18.04. Thank for your answer very much :tada:
